src/mono/mono/metadata/threads.c

```diff
@@ -1309,8 +1308,6 @@ create_thread (MonoThread *thread, MonoInternalThread *internal, MonoThreadStart
 		goto done;
 	}
 
-	internal->stack_size = (int) stack_set_size;
```

Comment:
nit: `mono_thread_platform_create_thread` for a bunch of platforms has code or comments that modifies the stack size argument to return the actual stack size of the new thread.  If we're not using that info anywhere anymore we could simplify `mono_thread_platform_create_thread`

(I assume we still have a way to query the stack size through other means - I think the Task subsystem needs to look it up, for example - instead of looking at the stack_size field on the thread object)